tests/aggregation/tests.py

```diff
@@ -395,6 +395,25 @@ def test_count(self):
         vals = Book.objects.aggregate(Count("rating", distinct=True))
         self.assertEqual(vals, {"rating__count": 4})
 
+    def test_annotate_values_count(self):
```

Comment:
I'd prefer a test name like `test_nonselected_annotations_not_in_groupby`, to clarify the purpose of the test.